assets/ar/factuality_disinformation_harmful_content/factuality/UnifiedFCFactuality_GPT4_ZeroShot.py

```diff
@@ -2,19 +2,19 @@
 import random
 import re
 
-from llmebench.datasets import FactualityUnifiedFCDataset
+from llmebench.datasets import UnifiedFCFactualityDataset
 from llmebench.models import GPTChatCompletionModel
-from llmebench.tasks import FactualityUnifiedFCTask
+from llmebench.tasks import UnifiedFCFactualityTask
 
 
 random.seed(1333)
 
 
 def config():
     return {
-        "dataset": FactualityUnifiedFCDataset,
+        "dataset": UnifiedFCFactualityDataset,
         "dataset_args": {},
-        "task": FactualityUnifiedFCTask,
+        "task": UnifiedFCFactualityTask,
```

Comment:
Should we unify and have only one Factuality task across Khouja/UnifiedFC?

Reply:
Yes but we need to return the needed measure with both Khouja/UnifiedFC assets, do you want to me to implement this change?

Reply:
Done in e812c9d